### ISSUE TITLE
Not connected

### ISSUE THREAD

**Issue by MyShlongIsLongAT** [evan-buss/openbooks#131]
![image](https://user-images.githubusercontent.com/72039636/216592207-72053c58-a708-439b-bcdd-61eaabd601df.png)
As you can see in this screenshot I get a timeout.
And I don't no why... Do I need some kind of port forward, so irc.highway.net can send me something back?
When I connect via mIRC i can join channels and everything works perfectly.

**Comment by Unrepentant-Atheist** [comment]
What was the solution?

**Comment by MyShlongIsLongAT** [comment]
Hello, @Unrepentant-Atheist,

Apologies for not including my solution earlier in this thread. It seems I made an oversight and opened a new issue unnecessarily, possibly because the original problem had already resolved itself.

In any case, I have managed to get everything working now by implementing the configuration mentioned in thread #132. As far as I can tell, the IRC simply enforces a temporary ban on new members for a brief period, but I'm not entirely certain, as it was discussed in another thread.

I hope my contribution proves helpful to you. If you have any further questions, feel free to ask. :)
